README.md

```diff
@@ -146,6 +146,34 @@ WMCO is not responsible for Windows operating system updates. The cluster admini
 creating the VMs and hence, the cluster administrator is responsible for providing an updated image. The cluster 
 administrator can provide an updated image by changing the image in the MachineSet spec.
 
+## Applying autoscaling to WMCO-managed Windows Machines
```

Comment:
How does this process differ from Linux Machines?

If it doesn't it might make more sense to just point to the proper docs so we have less documentation to maintain

Reply:
Great point... there is no difference between them. In that case would we want a short section that just points to the official docs, or delete this PR altogether?

Reply:
I think that comes down to the reason this PR is being created. Is this something that we feel we need to convey to customers? Basically, is this a common question that we are getting? If so lets have a short section pointing to the official docs. If not? The section might not be needed. But what about the case where we have Windows only customers who are using only using this readme/the csv for MachineSet configuration? A short section makes sense. All that considered I think having a short section is better choice.

The other thing we have to think about is whether or not we are properly covering this in our tests. What is the underlying mechanism that autoscaling needs? Is it `oc adm top`? If so maybe the better way to approach this is adding an e2e test for interacting with a Windows node with `oc adm top`. If we arent testing this can we really promote this functionality?

Reply:
Moved forward with a short section calling out our support + OpenShift docs.

Also, clarified the underlying mechanism [on the story](https://issues.redhat.com/browse/WINC-551?focusedCommentId=16205716&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16205716).